src/app/portlets/content-types/fields/content-type-fields-drop-zone/content-type-fields-drop-zone.component.spec.ts

```diff
@@ -3,13 +3,13 @@ import { DOTTestBed } from '../../../../test/dot-test-bed';
 import { DebugElement, Component, Input, SimpleChange, Output, EventEmitter, Injectable } from '@angular/core';
 import { ContentTypeFieldsDropZoneComponent } from './';
 import { By } from '@angular/platform-browser';
-import { Field, FieldRow, ContentTypeFieldsPropertiesFormComponent } from '../';
+import { Field, FieldRow } from '../';
```

Comment:
Make sure you clean up things you don't use... I know there are tons but make sure not introduce more. This was catched with tslint but with one update stop working, if anybody knows how to do it let me know... what I do what I do is that I open the files with WebStorm before pushing.